### PR TITLE
fix: upgrade emitter dependency to fix license issue

### LINKED ISSUE #11: Use renamed component-emitter which has license pkg json field
Right now this package depends on `emitter-component`, this package was renamed to `component-emitter` and the issue with the previous one is that it has a licensing issue, because there is no license field in its published pkg json on NPM, this causes unclear licensing issue for dependents which is a significant problem in enterprise environments.

Would it be possible to publish a new version of stream (0.0.3) with that component-emitter dependency upgraded to latest? I'm happy to open a PR

### PR DESCRIPTION
**Pull request by jorenbroekema**
fixes https://github.com/juliangruber/stream/issues/11

I wasn't sure how to run the makefile/tests, the make command gives me an error and it's not entirely clear to me how the `test.html` is ran exactly.

If needed I could modernize the test setup a little bit and ensure none of the tests broke with this dependency upgrade, or you can check things manually yourself, let me know.

### CODE REVIEW

**Comment by jorenbroekema** [comment]
just noticed https://github.com/juliangruber/stream/pull/6 and this PR fortunately also fixes that

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stream",
-  "version": "0.0.2",
+  "version": "0.0.3",
```
Comment (juliangruber) [inline package.json:3]: please revert the version upgrade, I will do that on my end

**Review by juliangruber — CHANGES_REQUESTED** [review]
otherwise lgtm!

File: package.json (hunk 1 above)
Comment (jorenbroekema) [reply package.json]: Reverted

File: package.json [hunk 2]
```diff
@@ -16,10 +16,10 @@
...
   "dependencies": {
-    "emitter-component": "^1.1.1"
+    "component-emitter": "^2.0.0"
   },
   "devDependencies": {
     "mocha": "*",
     "expect.js": "*"
   }
-}
+}
```
Comment (juliangruber) [inline package.json:25]:
```suggestion
}

```

**Review by juliangruber — CHANGES_REQUESTED** [review]

**Comment by juliangruber** [comment]
Thanks! Published as 0.0.3

**Pull request merged**